apps/builder/app/builder/features/style-panel/sections/backgrounds/background-layers.test.ts

```diff
@@ -0,0 +1,177 @@
+import {
+  layeredBackgroundProps,
+  setLayerProperty,
+  addLayer,
+} from "./background-layers";
+
+import { describe, test } from "@jest/globals";
+import type { StyleInfo } from "../../shared/style-info";
+import type { CreateBatchUpdate } from "../../shared/use-style-data";
+import type {
+  LayersValue,
+  StyleProperty,
+  StyleValue,
+} from "@webstudio-is/css-data";
+
+// @todo remove at node18
+globalThis.structuredClone = (v) => JSON.parse(JSON.stringify(v));
```

Comment:
value, not v

Reply:
please reread everything and remove abbreviations, a lot of them, not commenting each